announcement_4.6.md

```diff
@@ -0,0 +1,107 @@
+I'm excited to announce that Plotly.py 4.6 is now available for download via `pip` and `conda`! 
+For up-to-date installation instructions (including the **extra required steps for JupyterLab**!) please 
+see our [Getting Started](https://plot.ly/python/getting-started/) documentation page and
+if you run into trouble, check out our [Troubleshooting Guide](https://plot.ly/python/troubleshooting/).
+
+# What's new in Plotly.py 4.6
+
+Our [changelog](https://github.com/plotly/plotly.py/releases/tag/v4.6.0) has links to individual pull requests, 
+but here are the highlights:
+
+### Unified Hover Labels
+
+Until today hovering on plots resulted in either one more multiple hover labels, each always attached to a single point.
```

Comment:
one or multiple?

Reply:
hehe ok that sentence is correct I think but hard to parse. It's intended to be read as `Until today hovering on plots resulted in either (one more multiple hover labels), (each always attached to a single point).`